Gemfile.lock

```diff
@@ -387,6 +387,7 @@ GEM
 
 PLATFORMS
   ruby
+  x86_64-darwin-16
```

Comment:
Do we want this "darwin" platform line?

I don't really know what it does, but I believe it's indicating the gems can be installed directly on a macOS machine, rather than, say, the Heroku instance (Linux) or the Vagrant instance we test with (also Linux).

If I manually add this line in my `Gemfile.lock`, I get an error message when trying to run `bundle update`:

```
Could not find gem 'rails (= 5.1.4) x86_64-darwin-16' in any of the gem sources
listed in your Gemfile or available on this machine.
```

Probably best to just edit this line out if it's still there at merge time.